src/sentry/static/sentry/app/views/settings/projectAlerts/ruleEditor/index.jsx

```diff
@@ -101,9 +104,54 @@ const RuleEditor = createReactClass({
     Promise.all(promises).then(([environments, rule]) => {
       this.setState({environments, rule});
     });
-  },
+  }
+
+  fetchStatus() {
+    this.pollHandler();
+    this.startPolling();
+  }
+
+  pollHandler = async () => {
+    const {api, organization, project} = this.props;
+    const {uuid} = this.state;
+    const origRule = {...this.state.rule};
+
+    const {status, rule} = await api.requestPromise(
```

Comment:
@MeredithAnya We probably need a try/catch for the polling here in case we don't get an error here. If we get an error, we need to show the usual error message and stop the polling.